apps/tangle-dapp/app/account/Actions.tsx

```diff
@@ -66,6 +83,16 @@ const Actions: FC = () => {
           Icon={CoinsLineIcon}
         />
 
+        {isPayoutsAvailable && (
+          <ActionItem
+            hasNotificationDot
+            label="Payouts"
+            icon={<CoinsLineIcon size="lg" />}
+            internalHref={AnchorPath.NominationAndPayouts}
+            tooltip="You have payouts available. Click here to visit the Payouts page."
+          />
+        )}
```

Comment:
I think this is a good idea; I didn't consider the fact that payouts aren't always available to everyone.